build.sbt

```diff
@@ -223,6 +223,32 @@ lazy val client = (project in file("target/clients/java"))
     }
   )
 
+lazy val pythonClient = (project in file("clients/python/target"))
+  .enablePlugins(OpenApiGeneratorPlugin)
+  .settings(
+    // name of the generation step. See `openApiPackageName` for the actual Python package name
+    name := s"$artifactNamePrefix-python-client",
+    commonSettings,
+    (Compile / compile) := ((Compile / compile) dependsOn generate).value,
+
+    // OpenAPI generation specs
+    openApiInputSpec := (file(".") / "api" / "all.yaml").toString,
+    openApiGeneratorName := "python",
+    openApiOutputDir := (file("clients") / "python" / "target").toString,
```

Comment:
now that i think about it, this does not need to be named target at all. if we were not generating this on the fly, rather checking in the code, we would have named this directory `src` or something like that right? lets do that?

Reply:
The java client had a similar debate. We used to have it under src and commit to the repo but it cluttered the repo quite a bit and we ultimately decided to instead move to target since it was generated code that could be always generated on the fly. Having this generated code in the repo shows up as additional changes in PR etc and doesn't really add any value by actually being present in the repo imo.

Reply:
yeah. i agree with not having the code checked in. that's not my proposal at all. I am just saying that do we need to generate the code inside a directory named target which typically used for generated binaries? 

either ways, i should have mentioned.. this is a nit. i dont have a strong opinion, and can be changed later as well easily (assuming dev instructions in README dont have to change for this).